test/cql-pytest/test_cast_data.py

```diff
@@ -100,6 +100,21 @@ def test_cast_from_large_varint_to_varchar(cql, table1, cassandra_bug):
     cql.execute(f'INSERT INTO {table1} (p, cVarint) VALUES ({p}, {v})')
     assert [(str(v),)] == list(cql.execute(f"SELECT CAST(cVarint AS varchar) FROM {table1} WHERE p={p}"))
 
+# In test_cast_column_names we checked that columns are named in Cassandra style
+def test_cast_column_names(cql, table1, cassandra_bug):    
```

Comment:
Did you put here cassandra_bug by mistake?
cassandra_bug means that this test is known to fail on Cassandra. This shouldn't be the case here - the whole point of this issue was to make Scylla compatible with Cassandra, so the test for it should pass on Cassandra.
Please remove the "cassandra_bug" from this and the following test, and check that it works on Cassandra (`test/cql-pytest/run-cassandra test_cast_data.py::test_cast_column_names`).